x11-misc/util_macros/util_macros-1.19.2.recipe

```diff
@@ -6,9 +6,9 @@ HOMEPAGE="https://cgit.freedesktop.org/xorg/util/macros/"
 COPYRIGHT="2005 Red Hat, Inc.
 	2005, 2006 Oracle and/or its affiliates"
 LICENSE="MIT"
-REVISION="2"
+REVISION="1"
 SOURCE_URI="http://xorg.freedesktop.org/archive/individual/util/util-macros-$portVersion.tar.bz2"
-CHECKSUM_SHA256="18d459400558f4ea99527bc9786c033965a3db45bf4c6a32eefdc07aa9e306a6"
+CHECKSUM_SHA256="d7e43376ad220411499a79735020f9d145fdc159284867e99467e0d771f3e712"
 SOURCE_DIR="util-macros-$portVersion"
 
 ARCHITECTURES="x86_gcc2 x86 x86_64"
```

Comment:
This should be changed to `any`, since the package doesn't have architecture dependant bits.

Reply:
@alaviss this works ok for util_macros, but not for xproto and fontsproto, should I still use "any" in this case then?